### PR TITLE
Core/Packets: Fix calendar event time

### LINKED ISSUE #1287: Error time events in the calendar.
##### SMALL DESCRIPTION:

If you create a guild event or an event for a player, then the set time is reduced by 3 hours.

##### STEPS TO REPRODUCE THE PROBLEM:

1. Open calendar
2. Create event
3. Select a time (for example, 10:00 AM) and click save.
4. Click on the event created and see the selected time.
If you put 10:00 AM will show 07:00 AM

##### BRANCH(ES):

master

##### OPERATING SYSTEM:

Windows 10 x64

##### MODULES:

clear

### PR DESCRIPTION
**Pull request by brussens**
##### CHANGES PROPOSED:

-  
-  

###### ISSUES ADDRESSED:

Closes #1287

##### HOW TO TEST THE CHANGES:

1. Open calendar
2. Create event
3. Select a time (for example, 10:00 AM) and click save.
4. Click on the event created and see the selected time.
5. Should be 10:00 AM there too

##### KNOWN ISSUES AND TODO LIST:

- [ ]
- [ ] 

##### Target branch(es):

Master

### CODE REVIEW

**Comment by brussens** [comment]
Test this please on other timezones.

**Comment by FrancescoBorzi** [comment]
@brussens  can you please fill `HOW TO TEST THE CHANGES:` ?

File: src/common/Packets/ByteBuffer.h [hunk 1]
```diff
@@ -365,9 +365,9 @@ class ByteBuffer
             lt.tm_year = ((packedDate >> 24) & 0x1F) + 100;
 
             #ifdef OS_WIN
-                return uint32(mktime(&lt) + _timezone);
+                return uint32(mktime(&lt));
             #else
-                return uint32(mktime(&lt) + timezone);
+                return uint32(mktime(&lt));
             #endif
```
Comment (FrancescoBorzi) [inline src/common/Packets/ByteBuffer.h]: do we still need the `#ifdef` block ? the content is the same

**Comment by poszer** [comment]
> @brussens can you please fill `HOW TO TEST THE CHANGES:` ?

Done

**Comment by FrancescoBorzi** [comment]
@poszer thanks a lot

File: src/common/Packets/ByteBuffer.h (hunk 1 above)
Comment (BarbzYHOOL) [reply src/common/Packets/ByteBuffer.h]: no, it's useless, need to edit it

**Comment by FrancescoBorzi** [comment]
thank you! waiting for tester

**Comment by BarbzYHOOL** [comment]
@brussens thank you!!

**Comment by poszer** [comment]
Tested in game, without this update if you create Event or Guild Event for example tonight in 23:00, and when you open calendar to check events, it will show wrong time, like event is starting in 20:00 even when you created it in 23:00 which is **incorrect**.
Tested with this update on 4 Characters (Guild Members), when i create event/guild event in 23:00 it show **correct** time in calendar too.  
@brussens  Thank you :+1:

**Comment by FrancescoBorzi** [comment]
well done @brussens ! thanks @poszer for testing it!

**Pull request merged**